helm-charts/templates/control-plane/management-server/management-server-deployment.yaml

```diff
@@ -59,15 +75,26 @@ spec:
             - containerPort: 8766
       volumes:
         - name: management-server-keystore-secret-volume
-          secret: 
-            secretName: {{ template "apk-helm.resource.prefix" . }}-management-server-cert
-        - name: adapter-truststore-secret-volume
           secret:
-          {{- if and .Values.wso2.apk.dp.adapter.configs .Values.wso2.apk.dp.adapter.configs.tls }}
-            secretName: {{ .Values.wso2.apk.dp.adapter.configs.tls.certificatesSecret | default (printf "%s-adapter-server-cert" (include "apk-helm.resource.prefix" .)) }}
+          {{- if and .Values.wso2.apk.cp.managementServer.configs .Values.wso2.apk.cp.managementServer.configs.tls }}
+            secretName: {{ .Values.wso2.apk.cp.managementServer.configs.tls.certificatesSecret | default (printf "%s-management-server-cert" (include "apk-helm.resource.prefix" .)) }}
           {{- else }}
-            secretName: {{ template "apk-helm.resource.prefix" . }}-adapter-server-cert
+            secretName: {{ template "apk-helm.resource.prefix" . }}-management-server-cert
           {{- end }}
+        - name: adapter-truststore-secret-volume
+          secret:
+          {{ if and .Values.wso2.apk.dp.adapter.configs .Values.wso2.apk.dp.adapter.configs.tls }}
+            secretName: {{ .Values.wso2.apk.dp.adapter.configs.tls.certificatesSecret | default "apk-root-certificate"}}
+          {{ else }}
+            secretName: apk-root-certificate
```

Comment:
this root certificate for validating certs when connecting to multiple adaptors?

Reply:
yes